<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_listSTbyPhong</class>
 <widget class="QWidget" name="Form_listSTbyPhong">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Danh sách phòng</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView_phong"/>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Danh sách SV của phòng</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="treeView_sv"/>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Thoát</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>In</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form_listSTbyPhong</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
